types/depcheck/depcheck-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as depcheck from 'depcheck';
```

Comment:
Since `depcheck` is function, it should be `import depcheck = require('depcheck');` here